src/StdStorage.sol

```diff
@@ -68,16 +68,21 @@ library stdStorageSafe {
                 if (prev == bytes32(0)) {
                     emit WARNING_UninitedSlot(who, uint256(reads[i]));
                 }
+                if (prev != fdat) {
+                    continue;
+                }
+                bytes32 newVal = bytes32(~uint256(prev));
                 // store
-                vm.store(who, reads[i], bytes32(hex"1337"));
+                vm.store(who, reads[i], newVal);
                 bool success;
-                bytes memory rdat;
+                bytes32 dat;
```

Comment:
Wondering if these changes were required for any reason too? Based on the PR description I'd have thought the only diff is change from testing `hex"1337"` to testing `~prev`

Reply:
I was thinking of a very edge case that the `fdat` variable would be changed to something new but not equal to `~prev`, and the following steps in the loop would be affected since there's a `prev ~= fdat` check.

But it'll be a very weird case if this happens (function return value depends on multiple slot values), I'll just revert this back.